### PR TITLE
Added git-cola profile

### PR DESCRIPTION
**Pull request by kortewegdevries**

### CODE REVIEW

File: etc/profile-a-l/git-cola.profile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+protocol unix,inet,inet6,netlink
+seccomp
+shell none
+tracelog
+
+# private-bin atom,bash,colordiff,emacs,fldiff,geany,gedit,git,git gui,git-cola,git-dag,gitk,gpg,gvim,leafpad,meld,mousepad,nano,notepadqq,python*,sh,ssh,vim,vimdiff,which,xed
+private-cache
+private-dev
+# Comment if you sign commits with GPG
+private-etc ca-certificates,crypto-policies,fonts,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,localtime,login.defs,machine-id,mime.types,nsswitch.conf,passwd,resolv.conf,selinux,ssl,X11,xdg
```
Comment (rusty-snake) [inline etc/profile-a-l/git-cola.profile:55]: add `alternatives,pki`

Comment (kortewegdevries) [reply etc/profile-a-l/git-cola.profile]: Thanks!

File: etc/profile-a-l/git-cola.profile [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
```
Comment (Fred-Barclay) [inline etc/profile-a-l/git-cola.profile:28]: include disable-xdg.inc ?? How likely is it a user would need access to Documents, Music, Pictures, or Videos?

Comment (kortewegdevries) [reply etc/profile-a-l/git-cola.profile:28]: Thanks. I was testing with files in Documents and forgot to add it.

**Comment by Fred-Barclay** [comment]
Merged, thanks!

**Pull request merged**